<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueRedAuth</class>
 <widget class="QDialog" name="QueueRedAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #DCDCDC;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_auth">
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Авторизация</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_login">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(245, 245, 245,50%);
border: 1 solid grey;
border-radius: 5;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_password">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(245, 245, 245,50%);
border: 1 solid grey;
border-radius: 5;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_auth">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: rgba(245, 245, 245,100%); border: 1 solid grey; border-radius: 5;}
QPushButton:hover{background: grey; border: 1 solid grey; border-radius: 5;} </string>
     </property>
     <property name="text">
      <string>Авторизация</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labe_status">
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
